guides/doc-Configuring_Load_Balancer/topics/Overview.adoc

```diff
@@ -28,7 +28,7 @@ Load balancer works with the following services and features:
 * Content Management with `yum` repositories
 * Optional: Puppet
 
-NOTE: Provisioning hosts is not supported in the {Project} load-balanced setup.
+NOTE: The Load Balanced setup offers high availability for accessing yum or Puppet content. Other services running on the individual {SmartProxyServer}s such as provisioning, virt-who etc should not be accessed through the Load Balancer but can be accessed directly from the {SmartProxy}.
```

Comment:
```suggestion
NOTE: In the load-balanced setup, a load balancer distributes load only for the services and features mentioned above. You must access other services running on the individual {SmartProxies}, such as provisioning and virt-who, directly through {SmartProxies} and not through the load balancer.
```
Hi @ckyriakidou, thank you for your clarification on the topic. I have changed your writing a bit to add clarity. Let me know what you think.

Reply:
@spetrosi  How about "In the load-balanced setup, a load balancer distributes load only for the services and features mentioned above. If other services such as provisioning, or virt-who are running on the individual {SmartProxies}, you must access them directly through {SmartProxies} and not through the load balancer." So that it shows it's not obligatory to have other services on the LB capsule.

Reply:
Sounds perfect!